0100-private-projects.md

```diff
@@ -0,0 +1,36 @@
+- Feature Name: Private Projects
+- Created: 2020-02-10
```

Comment:
Is this information needed, given that we're storing everything in the repository?

Reply:
I guess it's nice to have that there, instead of having to look at git history?